src/upload/catalog/controller/extension/payment/libwebpay_rest/HealthCheck.php

```diff
@@ -47,7 +49,18 @@ private function getValidatephp(){
         }
         return $this->versioninfo;
     }
-
+    private function getOpenCartVersion() {
+        if (file_exists($this->configFile)) {
+            $fileContent = file_get_contents($this->configFile);
+            if (preg_match("/define\('VERSION', '([^']+)'\);/", $fileContent, $matches)) {
+                return $matches[1];
+            } else {
+                throw new Exception('No se pudo encontrar la versión de OpenCart.');
+            }
+        } else {
+            throw new Exception('El archivo admin/index.php no existe.');
```

Comment:
are this exceptions controlled when this method is used? maybe can return an empty or 'not found' string instead exceptions